src/Stripes.js

```diff
@@ -128,6 +131,15 @@ class Stripes {
   clone(extraProps) {
     return new Stripes(Object.assign({}, this, extraProps));
   }
+
+  /**
+   * reset
+   * reset config and logger properties to their default values
+   */
+  reset() {
+    this.config = config;
```

Comment:
@zburke I think @MikeTaylor is right here. I was not aware that the original `config` can be modified. If that's the case do you think we should clone it when the original stripes object is created here?

https://github.com/folio-org/stripes-core/blob/a9ca1c2ac8a7df97be4ad436d460709b100605a1/src/components/Root/Root.js#L128

Otherwise, we may still have a problem similar to:

https://jsbin.com/zoxiperuva/edit?js,console

Reply:
Unfortunately, @MikeTaylor is absolutely correct. My first crack at doing a deep clone when initializing stripes didn't work however. I'll spend a few more minutes on this, but the likely short-term outcome is that I'll unassign the ticket and leave it open.

Reply:
What, after all the work we've put into it already? :-)

I urge you to consider the point made on the Jira. If after thinking that through you remain convinced that this is the right thing to do, then I think we should put in the extra effort to get this over the line.

Reply:
I agree with @MikeTaylor! @zburke this all looks great. I think we just have to make sure we don't mutate the original config and it will be ready to go.

Reply:
… assuming we want to do it at all, of course! See https://issues.folio.org/browse/STCOR-674#:~:text=added%20a%20comment%20%2D-,Yesterday,-I%27m%20not%20sure

Reply:
@MikeTaylor I missed your comment and also didn't realize that the mutation only happens via ui-developer. Hmm... I will leave it to you two to decide what to do here. I kind if like what Zak did here and the additional tests too.

Reply:
Even if we don't keep the change in functionality, we should definitely rescue the new tests.